pull/src/handler.cc

```diff
@@ -151,7 +151,7 @@ bool MetricsHandler::handleGet(CivetServer*, struct mg_connection* conn) {
     metrics = CollectMetrics(collectables_);
   }
 
-  auto serializer = std::unique_ptr<Serializer>{new TextSerializer()};
+  auto serializer = detail::make_unique<TextSerializer>();
```

Comment:
There's no need for the Text Serializer to be created on the heap. Please create an instance on the stack.